bee-node/bee-node/src/workers.rs

```diff
@@ -19,20 +19,20 @@ const DEFAULT_BIND_ADDRESS: &str = "0.0.0.0:3030";
 #[must_use]
 pub struct MetricsConfigBuilder {
     #[serde(skip)]
-    pid: Option<u32>,
+    pid: u32,
```

Comment:
Since this is now mandatory I'd rather have a `new` method that take it tbh. Relying on default is a bit weird.

Reply:
I ended up changing things a bit, let me know what you think. I didn't like the default behaviour of the pid.